doc/man7/store.pod

```diff
@@ -0,0 +1,104 @@
+=pod
+
+=head1 NAME
+
+store - Store retrieval functions
```

Comment:
If you changed it to "osslstore - Store retrieval functions" then yes.  The filename should match ojne of the entries in the NAME section.  The find-doc-nits script should have complained.